DAO.sol

```diff
@@ -500,8 +500,18 @@ contract DAO is DAOInterface, Token, TokenSale {
 
         uint quorum = p.yea + p.nay;
 
+        // require 53% for calling newContract()
+        bool updateContractQuorumCheck = true;
+        if (_transactionData.length >= 4 && _transactionData[0] == 0x68
+            && _transactionData[1] == 0x37 && _transactionData[2] == 0xff
+            && _transactionData[3] == 0x1e
```

Comment:
This is the hash signature of `newContract()`. The comment explains what it is quite well.
To obtain it just run `solc --hashes DAO.sol`